File: media_numero.rb
class comparador
  def initialize(a, b, c)
    @a = a
    @b = b
    @c = c
    
  end

  def comparar_ab
    @a > @b
end

def mostrar_resultado
  puts comparar_ab
end
end

# Uso
comp = comparador.new(1, 2, 3)
comp.mostrar_resultado # Irá imprimir false

File: media_numero.js
class Comparador {
    constructor(a, b, c) {
        this.a = a;
        this.b = b;
        this.c = c;
    }

    compararAB() {
        if (this.a > this.b) {
            return this.a;
        } else {
            return this.b;
        }
    }

    mostrarResultado() {
        console.log(this.compararAB());
    }
}

// Uso
const comparador = new Comparador(5, 10, 15);
comparador.mostrarResultado(); // irá imprimir: false